### PR TITLE
Update CircleCI to use cuda11

### PR DESCRIPTION
**Pull request by wonjoolee95**
Draft PR to update CircleCI to use cuda11. Posting draft PR to see if CircleCI tests will fail.

### CODE REVIEW

**Comment by JackCaoG** [comment]
You might want to update **https://github.com/pytorch/xla/blob/cfe4711d58b6e26c0fab84a85cc83f755a8c0e87/.circleci/common.sh#L93** Not sure if gcc7 will work. Worse case we can try to install clang-9 or find a different docker.

**Review by JackCaoG — APPROVED** [review]
Nice!

**Comment by JackCaoG** [comment]
@wonjoolee95 We can work on updating tf now

**Comment by wonjoolee95** [comment]
> @wonjoolee95 We can work on updating tf now

Thanks! Sounds good, just created a tracking issue -- https://github.com/pytorch/xla/issues/3267.

File: .circleci/common.sh [hunk 1]
```diff
@@ -90,7 +94,7 @@ function install_deps_pytorch_xla() {
 function build_torch_xla() {
   XLA_DIR=$1
   pushd "$XLA_DIR"
-  CC=clang-9 CXX=clang++-9 python setup.py install
+  CC=gcc-7 CXX=gcc-7 python setup.py install
```
Comment (seemethere) [inline .circleci/common.sh:97]: This change actually broke upstream CI for `pytorch/pytorch`, see https://github.com/pytorch/pytorch/runs/4651120454?check_suite_focus=true

What was the reasoning behind the compiler switch?

Comment (JackCaoG) [reply .circleci/common.sh:97]: @wonjoolee95 Can we use clang-9 for all of the compilation task for pt/xla?

**Pull request merged**